### PR TITLE
refactor: add `get_web_token` to `WebApi`

### PR DESCRIPTION
**Pull request by tamssokari**
- [x] Add `get_web_token` method to `WebApi`
- [x] Add tests for `get_web_token` method
- [x] Add docs for `get_web_token` method

### CODE REVIEW

File: src/web-api.js [hunk 1]
```diff
@@ -375,6 +375,75 @@ class WebApi {
...
+			['user_id', 'job_id', 'product'].forEach(requiredParam => {
+				if (!requestParams[requiredParam]) {
+					reject( new Error(`${requiredParam} is required to get a web token`) );
+				}
+			});
+
+			const timestamp = new Date().toISOString();
+			const signature = new Signature(this.partner_id, this.api_key).generate_signature(timestamp).signature;
+
+			const body = JSON.stringify({
```
Comment (beaglebets) [inline src/web-api.js:397]: request is missing `callback_url`, which is a required field.

File: src/web-api.js [hunk 2]
```diff
@@ -375,6 +375,75 @@ class WebApi {
...
+		return new Promise((resolve, reject) => {
+			if (!requestParams) {
+				reject( new Error('Please ensure that you send through request params') );
+			}
+
+			if (typeof requestParams !== 'object') {
+				reject( new Error('Request params needs to be an object') );
+			}
+
+			['user_id', 'job_id', 'product'].forEach(requiredParam => {
```
Comment (beaglebets) [inline src/web-api.js:388]: You should check that either the class was instantiated with a callback url or that one was passed in request params also.

**Review by beaglebets — CHANGES_REQUESTED** [review]

File: src/web-api.js (hunk 2 above)
Comment (tamssokari) [reply src/web-api.js:388]: gotcha. thanks.

Comment (tamssokari) [reply src/web-api.js:388]: should that be repeated here though? considering it is done in the lambda?

Comment (tamssokari) [reply src/web-api.js:388]: doing so here will mean we pass an `options` object, with a `callback` entry. 

at least to follow convention

Comment (beaglebets) [reply src/web-api.js:388]: yes it should be here too. There is no point in allowing requests you know will fail. It only slows down the feedback cycle for those integrating. We validate in the lambda for requests that might not be coming from our library.

Comment (tamssokari) [reply src/web-api.js:392]: added the check for `callback_url` or `this.default_callback` above this

File: src/web-api.js (hunk 1 above)
Comment (tamssokari) [reply src/web-api.js:401]: fixed.

File: README.md [hunk 3]
```diff
@@ -426,6 +427,43 @@ Your response will return a promise that contains a JSON Object below (with imag
...
+
+```json
+{
+	// String: required
+	"user_id": 'user-1',
+	// String: required
+	"job_id": 'job-1',
+	// String: required one of 'authentication', 'identity_verification', 'smartselfie', 'ekyc_smartselfie', 'enhanced_kyc', 'document_verification'
+	"product": 'authentication',
+	// String: optional, defaults to global callback url
```
Comment (beaglebets) [inline README.md:450]:
```suggestion
	// String: required, optional if callback url was set during instantiation of the class
```

**Review by beaglebets — APPROVED** [review]

**Pull request merged**